Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -188,3 +188,15 @@
     layers:
       - state: box
       - state: writing
+
+- type: entity
+  parent: FoodBoxDonut
+  id: FoodBoxDonutFilled
+  suffix: Filled
+  components:
+  - type: StorageFill
+    contents:
+      - id: FoodDonutPlain
+        amount: 5
+      - id: FoodDonutPink
+        amount: 1
```

Comment:
What's the difference between this and its parent? it should already fill but with different amounts:
![image](https://user-images.githubusercontent.com/47410468/155063153-5d38f898-8621-4964-88c4-807d0f8d5b06.png)

Reply:
Oh my bad, i made this branch before this was a thing, fixed